BillSplitter/bill_splitter.py

```diff
@@ -54,4 +55,27 @@ def check_number(value):
 def meow():
     return 'meow'
 
-main()
+
+def add_tax(value):
+
+    location = input('What state are you in?: ')
+
+
+    state = ['Wyoming','Wisconsin','West Virginia','Washington','Virginia','Vermont','Utah', 'Texas','Tennessee','South Dakota','South Carolina','Rhode Island','Puerto Rico' ,'Pennsylvania','Oregon','Oklahoma','Ohio','North Dakota','North Carolina','New York','New Mexico','New Jersey','New Hampshire','Nevada','Nebraska','Montana','Missouri','Mississippi','Minnesota','Michigan','Massachusetts','Maryland','Maine','Louisiana','Kentucky','Kansas','Iowa','Indiana','Illinois','Idaho','Hawaii','Guam','Georgia','Florida','District of Columbia','Delaware','Connecticut','Colorado','California','Arkansas','Arizona','Alaska','Alabama'];
+
+    tax = ['0.06','0.0675','0.07','0.104','0.06','0.07','0.0835','0.0825','0.0975','0.06','0.09','0.07','0.115','0.08','0','0.11','0.08','0.08','0.075','0.08875','0.08688','0.12875','0','0.0825','0.075','0','0.1085','0.0725','0.07875','0.06','0.0625','0.06','0.055','0.12','0.06','0.1015','0.07','0.07','0.1025','0.085','0.04712','0.04','0.08','0.075','0.0575','0','0.0635','0.1','0.1025','0.11625','0.10725','0.07','0.135'];
+
+    tax_from_state = dict(zip(state,tax))
+
+    try:
+        tax_from_state[location]
+
+    except KeyError:
+        print('Not a valid state')
```

Comment:
you could ask for more input here and do this again, if you break this method into multiple methods this would be easier to do.

Reply:
Actually I think it does ask for input again at this point since it returns back to the beginning of the function

Reply:
that's true but you don't really _have_ to do this recursively.